openrtb_ext/bidders.go

```diff
@@ -126,6 +128,8 @@ var BidderMap = map[string]BidderName{
 	"visx":              BidderVisx,
 	"vrtcal":            BidderVrtcal,
 	"yieldmo":           BidderYieldmo,
+	"gumgum_ortb":		 BidderGumGumOrtb,
+	"pulsepoint_ortb":	 BidderPulsepointOrtb,
```

Comment:
Super minor formatting inconsistency in this file.  Maybe there's a good Go linter we can use for this sort of thing.

Even more nitpicky: the rest of the bidders seem to be in alphabetical order.  Worth preserving that with the new ones?